src/components/Fundraiser.tsx

```diff
@@ -393,28 +405,39 @@ export function FundraiserView(
                   </vstack>
                 </hstack>
                 <hstack width='34%' alignment='center middle' borderColor={DEBUG_MODE ? 'red' : 'neutral-border-weak'} border={DEBUG_MODE ? 'thin' : 'none'}>
-                  <FancyButton
-                    backgroundColor={everyGreen}
-                    textColor="white"
-                    height={40}
-                    icon="external"
-                    iconPosition="right"
-                    isExpanded={isButtonExpanded}
-                    onPress={() => {
-                      if (fundraiserInfo) {
-                        console.log("Navigating to fundraiser URL:", fundraiserURL);
-                        context.ui.navigateTo(fundraiserURL);
-                      }
-                    }}
-                  >
-                    Donate
-                  </FancyButton>
+                  {status === 'completed' || status === 'expired' ? (
```

Comment:
This condition should be a function/prop of FundraiserStatus as you'll be doing this a lot. e.g. <FundraiserStatus>.ended()